### ISSUE TITLE
Game crashes around 30min mark?

### ISSUE THREAD

**Issue by hoclun-rigsep** [FAForever/faf-linux#6]
Probably nothing to do with the scripts in this repo but I thought I would ask anyway: seems the game crashes roughly around 30 min for me, which is starting to annoy teammates :)

I don't see anything in game or client logs that suggests a cause to me, but I may not know what I should be looking for.

**Comment by iczero** [comment]
This is usually the result of FA (the game) running out of memory. This happens more often on modded games and Nvidia. There is no good fix for this currently, all the workarounds I have are already in the repo. Sorry.

**Comment by tatsujb** [comment]
yes, this is an issue that is unrelated to this script. as iczero pointed out things that can help eliminate this issue are: having an AMD gpu, (and definitely not an intel one), having a more up to date graphics driver, not using mods and using windowed mode on FA if at all possible.

**Comment by hoclun-rigsep** [comment]
I would like to report that I have not experienced this issue since switching from kwin to fluxbox.

I do use an Nvidia GPU and the most recent stable nvidia binary driver.

**Comment by veqqq** [comment]
> yes, this is an issue that is unrelated to this script. as iczero pointed out things that can help eliminate this issue are: having an AMD gpu, (and definitely not an intel one), having a more up to date graphics driver, not using mods and using windowed mode on FA if at all possible.

@tatsujb What is the issue with the inbuilt Intel GPU?

**Comment by iczero** [comment]
New wine wow64 may make this less of an issue but not sure. Please reopen if it still happens.
